library/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM fedora
+
+RUN dnf install -y java-1.8.0-openjdk.x86_64
```

Comment:
I don't know if docker supplies any generalized command to install package, but I did not see this. So I still used yum or dnf.

For from image, I'll change it.